src/client/Directory.Build.props

```diff
@@ -117,6 +117,15 @@
     <EmbeddedResource Include="$(PathToMsalSources)\Properties\Microsoft.Identity.Client.rd.xml" />
   </ItemGroup>
 
+  <ItemGroup Label="Dependency overrides due to security vulnerabilities">
+    <!-- 4.3.0 has CVE-2019-0657 -->
+    <PackageReference Include="System.Private.Uri" Version="4.3.2" />
+    <!-- 4.1.3 has CVE-2018-0786 -->
+    <PackageReference Include="System.ServiceModel.Http" Version="4.5.3" Condition="'$(TargetFramework)' == '$(TargetFrameworkUap)'" />
```

Comment:
I bumped the version of these packages only in UAP (and have tested things still work). Theoretically we can upgrade all the frameworks, but only UWP brings this 4.1.3 version of the libraries. 

Frameworks have metapackages associated with them, so we do not normally see these packages referenced directly.